cabal.project

```diff
@@ -1,6 +1,6 @@
 index-state:
   -- Bump this if you need newer packages from Hackage
-  , hackage.haskell.org 2024-12-09T15:45:06Z
+  , hackage.haskell.org 2024-12-30T00:00:00Z
```

Comment:
Im dubious about that exact 00 timestamp. I think cabal will not enjoy it

Reply:
Cabal does not complain for me

Reply:
The warning was demoted to being shown only in verbose mode. If you pass `-v` you shall see something like "There is no index state exactly at 2024-12-30T00:00:00Z, falling back to previous state at XXXXX"